### PR TITLE
Re-add `setColor` api

### LINKED ISSUE #423: Re-add `setColor` api
We need to re-add this method using the new `color` field of the shared model

_Originally posted by @trungleduc in https://github.com/jupytercad/JupyterCAD/pull/422#discussion_r1761071728_

### PR DESCRIPTION
**Pull request by arjxn-py**
Shall close #423

### CODE REVIEW

File: python/jupytercad_lab/jupytercad_lab/notebook/cad_document.py [hunk 1]
```diff
@@ -95,6 +95,21 @@ def get_object(self, name: str) -> Optional["PythonJcadObject"]:
...
+        """
+        Retrieve the color of a shape by its name or index.
+
+        :param shape_id: The name or index of the shape.
+        :return: The color of the shape in hex format.
+        """
+        shape = self.get_object(shape_id)
+        if hasattr(shape, "parameters") and hasattr(shape.parameters, "Color"):
+            color = shape.parameters.Color
+            print(color)
```
Comment (trungleduc) [inline python/jupytercad_lab/jupytercad_lab/notebook/cad_document.py:108]:
```suggestion
```

**Comment by trungleduc** [comment]
The old API `set_color(self, object_name, color)` is still missing?

**Comment by arjxn-py** [comment]
> The old API `set_color(self, object_name, color)` is still missing?

Yes, i'll recreate a method for that

**Comment by arjxn-py** [comment]
Bot please update snapshots

File: python/jupytercad_lab/jupytercad_lab/notebook/cad_document.py [hunk 2]
```diff
@@ -665,7 +725,16 @@ def set_visible(self, name: str, value):
...
 
-        obj["visible"] = False
+        obj["visible"] = value
+
+    def set_color(self, name: str, value: str):
+        obj: Optional[Map] = self._get_yobject_by_name(name)
+
+        if obj is None:
+            raise RuntimeError(f"No object named {name}")
+        parameters = obj.get("parameters")
```
Comment (trungleduc) [inline python/jupytercad_lab/jupytercad_lab/notebook/cad_document.py:735]:
```suggestion
        parameters = obj.get("parameters", {})
```

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks!

**Pull request merged**